### ISSUE TITLE
bug: Display "Live Examples" contributions in MDN Profile

### ISSUE THREAD

**Issue by elharony** [mdn/interactive-examples#1258]
I've noticed it today when my #1257 got merged, and my contribution doesn't appear on my MDN Profile `Recent Docs Activity`'s section

I would like to add the `Live Examples` contributions to the `MDN Profile` as well as the normal contributions via `MDN's Editor` got added

**An expected concequence;** Any contributions made via `Live Examples` aren't visible at the bottom of the page, and the `Last updated by ...` won't be accurate at this case!

**Comment by wbamberg** [comment]
Thanks for your contribution @elharony . Yes, contributions to interactive examples are not credited in the MDN page itself, they are credited in this repo. The same applies to other places where the page integrates content from other sources, like compat data, spec data, and so on.

I don't think we are planning to change this, but if you wish I can revert your contribution to this example.

**Comment by elharony** [comment]
Hi @wbamberg !

Definitely, I don't wish to revert my contribution, I just noticed that `Live Examples` contributions and other contributions like `Spec Data` aren't displaying on the `MDN Profile`. So, I reported it, in case you didn't notice it! 😄 

But yeah, not I understand that you know about it, and there is no plan to integrate those contributions with the `Editing` contributions

Thanks for explaining this! 👍
